javaparser-core/src/main/java/com/github/javaparser/ast/expr/MethodCallExpr.java

```diff
@@ -93,9 +93,9 @@ public MethodCallExpr addArgument(Expression arg) {
 		v.visit(this, arg);
 	}
 
-	public List<Expression> getArgs() {
-        args = ensureNotNull(args);
-        return args;
+	public List<Expression> getArgsList() {
```

Comment:
As noted in the issue, we need to be consistent in args and arguments usage. This is not